app/src/main/java/chat/rocket/android/server/domain/PermissionsInteractor.kt

```diff
@@ -71,6 +68,28 @@ class PermissionsInteractor @Inject constructor(
         } == true || userHelper.isAdmin()
     }
 
+
+    fun canSeeTheAdminChannel(): Boolean {
```

Comment:
I think there is a type here, wouldn't it be `canSeeTheAdminPanel()`?

Reply:
Good catch! 
https://github.com/RocketChat/Rocket.Chat.Android/pull/1570/commits/07dfbcdc586441b574fb1bd59bcb11f35f129513